sct.py

```diff
@@ -1147,7 +1147,7 @@ def send_email(test_id=None, test_status=None, start_time=None, started_by=None,
     else:
         LOGGER.warning("Failed to find test directory for %s", test_id)
     job_name = os.environ.get('JOB_NAME', '')
-    if 'latency' in job_name or 'throughput' in job_name or 'perf' in job_name:
+    if (('latency' in job_name or 'throughput' in job_name or 'perf' in job_name) and 'sla' not in job_name):
```

Comment:
@fgelcer we need to make this work base on the configuration, we can't read the SCT configuration at this stage ?
doing logic base on the name of tests is problematic as can be seen here. (I guess it won't be the last time)

Reply:
Agree. We can, for example, add parameter, like "test_type": longevity/performance... And set it in the relevant groovy

Reply:
added this to an existing task, to make improvements in this area